packages/@aws-cdk/aws-stepfunctions-tasks/lib/dynamodb/shared-types.ts

```diff
@@ -232,6 +232,14 @@ export class DynamoAttributeValue {
     return new DynamoAttributeValue({ BOOL: value });
   }
 
+  /**
+   * maps a boolean from a string - can be a literal or a json path
+   * @param value the string to map
+   */
+  public static booleanFromJsonPath(value: string) {
```

Comment:
there should be a unit test added for this method (happy path && Json path error throws)

Reply:
good